CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 * [#481](https://github.com/clojure-emacs/clojure-mode/issues/481): Support vertical alignment even in the presence of blank lines, with the new `clojure-align-separator` user option.
 * [#483](https://github.com/clojure-emacs/clojure-mode/issues/483): Support alignment for reader conditionals, with the new `clojure-align-reader-conditionals` user option.
+- Indent "let", "when" and "while" as function form if not at start of a symbol
```

Comment:
Please, make this entry consistent with the other entires in the changelog (e.g. link PR, use `*` instead of `-`, add `.`).

Reply:
Thanks, fixed the chicken-and-egg plus wrong list item type